src/main/java/com/iexec/common/security/CipherUtils.java

```diff
@@ -169,21 +173,21 @@ public static byte[] aesEncrypt(byte[] plainData, byte[] base64Key)
      * <p>
      * Note: PKCS5Padding is equivalent to PKCS7Padding
      * <p>
-     * Ref: https://stackoverflow.com/a/10194082/7631879
+     * Ref: <a href="https://stackoverflow.com/a/10194082/7631879">https://stackoverflow.com/a/10194082/7631879</a>
      *
      * @param plainData to encrypt
-     * @param key
-     * @param iv        initialization vector
+     * @param key       AES key
+     * @param iv        Initialization vector
      * @return Encrypted data
-     * @throws GeneralSecurityException
+     * @throws GeneralSecurityException if a security related error occurs.
```

Comment:
For consistency with message on decryption ?
```suggestion
     * @throws GeneralSecurityException if an errors occurs during AES encryption
```

Reply:
Resolved in [cc09d9b](https://github.com/iExecBlockchainComputing/iexec-common/pull/470/commits/cc09d9b58dd26d71e3c901ac02505ef4264cb657)

Reply:
Resolved in [cc09d9b](https://github.com/iExecBlockchainComputing/iexec-common/pull/470/commits/cc09d9b58dd26d71e3c901ac02505ef4264cb657)